adhocteam/Head-Start-TTADP
# Dev environment db migration fix

**Description of change**

On sandbox: Skip undoing seeds entirely, since undoing all migrations will drop all tables.
On dev: Run migration before undoing the seeds, so that new tables exist before we try to delete out of them.

**How to test**

Verify that deploys continue to work without migration failures. Check migration failures with `cf tasks tta-smarthub-dev` and the output of `cf logs tta-smarthub-dev` while a deploy is happening.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/255

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
